src/clj/game/core/abilities.clj

```diff
@@ -279,9 +279,9 @@
 ;;; Optional Ability
 (defn optional-ability
   "Shows a 'Yes/No' prompt and resolves the given ability if Yes is chosen."
-  ([state side card msg ability targets] (optional-ability state side (make-eid state) card msg ability targets))
-  ([state side eid card msg ability targets]
-   (show-prompt state side eid card msg ["Yes" "No"]
+  ([state side card a-msg ability targets] (optional-ability state side (make-eid state) card a-msg ability targets))
+  ([state side eid card a-msg ability targets]
+   (show-prompt state side eid card a-msg ["Yes" "No"]
```

Comment:
I don't remember what `msg` is here, if it's a 5-fn I'd call it `msg-fn` otherwise I'd call it `message`

Reply:
I'll verify!

Reply:
It can be both. What do you suggest?

Reply:
Probably just `message` then. Or maybe `prompt-msg` since you use it with `show-prompt`.